docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+
+### Portable Browsers
+
+To use a portable browser, specify the path to the browser executable file. For more information and examples, see:
+
+* Command line: [Portable Browsers](../../command-line-interface.md#portable-browsers)
+* API: [runner.browsers](../reference/api/runner/browsers.md)
+* Configuration file: [browsers](../reference/configuration-file.md#browsers)
+
+### Browsers on Remote Devices
+
+To run tests on a remote mobile and desktop device, this device must have network access to the TestCafe server.
+
+First, you will need to create a remote browser connection.
+
+* Command line: specify the `remote` *alias* (see [Remote Browsers](../reference/command-line-interface.md#remote-browsers))
+* API: use the [createBrowserConnection](../reference/api/testcafe/createbrowserconnection.md) method
+* Configuration file: [browsers](../reference/configuration-file.md#browsers)
+
+After that, TestCafe will provide a URL to open on the remote device in the browser against which you want to test.
+As you open this URL, the browser connects to the TestCafe server and starts testing.
+
+> You cannot [take screenshots](../guides/basic-guides/interact-with-the-page.md#take-screenshot) or [resize the browser window](../guides/basic-guides/interact-with-the-page.md#resize-window) when you run tests in remote browsers.
+
+### Browsers in Cloud Testing Services
+
+TestCafe allows you to use browsers from cloud testing services. You can access them through [browser provider plugins](../extending-testcafe/browser-provider-plugin/README.md).
+
+The following plugins for cloud services are currently provided by the TestCafe team.
+
+Service                              | Plugin
+------------------------------------ | -------------------
+[Sauce Labs](https://saucelabs.com/)          | [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs)
+[BrowserStack](https://www.browserstack.com/) | [testcafe-browser-provider-browserstack](https://www.npmjs.com/package/testcafe-browser-provider-browserstack)
+
+You can search npm for plugins developed by the community. Their names begin with the `testcafe-browser-provider-` prefix: [https://www.npmjs.com/search?q=testcafe-browser-provider](https://www.npmjs.com/search?q=testcafe-browser-provider).
+
+You can also create your own plugin. See [Browser Provider Plugin](../extending-testcafe/browser-provider-plugin/README.md) for instructions.
+
+### Microsoft Edge Legacy Support
+
+Tests are run in Microsoft Edge Legacy if it is the only version of Edge installed on the machine.
+
+If Chromium-based and Legacy versions are available, you can change the default Edge application in **System Settings** to test in the Legacy version.
+
+To do this, open **Default apps** in **Windows Settings** and scroll down to select the option **Default Apps by Protocol**. Set the `MICROSOFT-EDGE:` protocol to Legacy Edge.
```

Comment:
Change the first sentence to:
To do this, open **Default apps** in **Windows Settings** and scroll down to select **Default Apps by Protocol**.